js/main.js

```diff
@@ -1,330 +1,388 @@
+        tab,
+        query: selectors.prevBtn,
+      },
+      callback,
+    );
   },
 
-  next: function (tab, callback) {
-    Spotify.click({
-      tab: tab,
-      query: '.nowPlayingBar-container .player-controls button[class^="control-button spoticon-skip-forward"]'
-    }, callback)
-  }
-}
+  next(tab, callback) {
+    Spotify.click(
+      {
+        tab,
+        query: selectors.nextBtn,
+      },
+      callback,
+    );
+  },
+};
 
-var App = {
-  setTheme: function (theme) {
-    var bodyColor = theme === 'light'
-      ? 'Dark'
-      : 'Light'
+const App = {
+  setTheme(theme) {
+    const bodyColor = theme === 'light' ? 'Dark' : 'Light';
 
-    document.body.setAttribute('class', theme)
-    findEl('#color-body').innerHTML = bodyColor
-  }
-}
+    document.body.setAttribute('class', theme);
+    findEl('#color-body').innerHTML = bodyColor;
+  },
+};
 
-function changeColor () {
+function changeColor() {
   if (document.body.className === 'light') {
-    document.body.setAttribute('class', 'dark')
-    document.getElementById('color-body').innerHTML = 'Light'
-    chrome.storage.sync.set({'color': 'dark'})
+    document.body.setAttribute('class', 'dark');
+    document.getElementById('color-body').innerHTML = 'Light';
+    chrome.storage.sync.set({ color: 'dark' });
   } else {
-    document.body.setAttribute('class', 'light')
-    document.getElementById('color-body').innerHTML = 'Dark'
-    chrome.storage.sync.set({'color': 'light'})
+    document.body.setAttribute('class', 'light');
+    document.getElementById('color-body').innerHTML = 'Dark';
+    chrome.storage.sync.set({ color: 'light' });
   }
 }
 
-function fetchLyrics () {
-  // get artist and trackname
-  var artist = findEl('#current-track-artist').innerHTML
-  var track = findEl('#current-track-name').innerHTML
-  var xhr = new XMLHttpRequest()
+function updateLyricsText(text) {
+  const container = findEl('#container-lyrics');
+  container.style.display = 'flex';
+  container.style.textAlign = 'justify';
+  container.innerText = text;
+}
 
-  xhr.onreadystatechange = function () {
-    if (xhr.readyState === 4 && xhr.status === 200) {
-      var response = JSON.parse(xhr.responseText)
+async function fetchLyrics() {
+  // get artist and trackname
+  const artist = findEl('#current-track-artist').innerHTML;
+  const track = findEl('#current-track-name').innerHTML;
+
+  updateLyricsText('Loading...');
+
+  // to protect accesstoken there is a custom backend
+  // see https://gitlab.com/lodi-g/spotify-control-chrome-extension-backend
+  const res = await fetch(BACKEND_URL, {
+    method: 'POST',
+    body: JSON.stringify({ artist, track }),
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  });
+
+  if (res.status === 404) {
+    updateLyricsText('Song not found on genius.com');
+  } else if (!res.ok) {
+    updateLyricsText(
+      'Unknown error, feel free to open an issue on GitHub: https://github.com/lodi-g/spotify-control-chrome-extension',
```

Comment:
hey, I'm glad to get this merged and then release a new version of the extension asap... can you please update the references from your github to mine across your changes? I'd like to keep all issues at the same place.